rosette/src/Export.cc

```diff
@@ -575,17 +621,166 @@ void rblTableObjectHandler( ObjectCodePB::Object * lvob, pOb ob, ObjectCodePB::O
     populateObjectByType(rt->tbl, tbl);
 }
 
+void ctxtObjectHandler( ObjectCodePB::Object * pbob, pOb ob, ObjectCodePB::ObType pbtype) {
```

Comment:
What was the reason again for exporting `Ctxt`?

Reply:
I had thought that a ctxt was required for execution of a Code block in the VM.  I may be mistaken.

Reply:
Yes, that's correct. However I don't think there is a need to export it (especially for the `opLookup` hack we want to do). The `Code` that runs inside of a certain `Ctxt` will change certain fields in the `Ctxt` while executing. The `Ctxt` is just there to store temporary state for the associated `Code` object.

Reply:
Done